### ISSUE TITLE
golangci-lint job is failing

### ISSUE THREAD

**Issue by davecheney** [projectcontour/contour#2340]
```
$ make lint
Running Go linter ...
Unable to find image 'golangci/golangci-lint:latest' locally
latest: Pulling from golangci/golangci-lint

Status: Downloaded newer image for golangci/golangci-lint:latest
fatal error: runtime: out of memory
```

golangci-lint job is exploding in CI.

**Comment by davecheney** [comment]
The recently release 1.24.0 release includes an update to static check, https://github.com/golangci/golangci-lint/releases/tag/v1.24.0, this might be causing the OOM

**Comment by davecheney** [comment]
https://github.com/golangci/golangci-lint/issues/994

**Comment by jpeach** [comment]
Fixed in #2341
